### PR TITLE
mrc-2013 Remove khttp dependency from OrderlyServerAPI

### PR DESCRIPTION
**Pull request by mwoodbri**
This removes all khttp usage from non-test OrderlyWeb code, standardising on OkHttp and enabling mrc-1908 to be implemented more cleanly

### CODE REVIEW

File: src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt [hunk 1]
```diff
@@ -119,23 +144,12 @@ class OrderlyServer(private val config: Config,
...
-        {
-            "?" + queryString
-        }
-        else
-        {
-            ""
-        }
-        return urlBase + url + queryPart
-    }
+    private fun buildFullUrl(url: String, queryString: String?) = "${urlBase}${url}?${queryString ?: ""}"
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt:154]: Much neater, thanks.

**Review by EmmaLRussell — APPROVED** [review]
This looks great thanks. 

I'm in two minds about whether there's any point in trying to reduce some of the request building/executing call boilerplate in OrderlyServerAPI, as there's a bit of repetition of that for each verb. Could have a private 'build request and execute call' method to which you pass parameters including the http verb you want. But that might just be replacing a small amount of repetition with something that might not be flexible enough to accommodate future changes. 

Up to you. Put something like that in if you think there's anything in it, or go ahead and merge if not!

**Comment by mwoodbri** [comment]
@EmmaLRussell Thanks! I did try to something along those lines: first by using a shared, part-completed `Builder` instance and then by repurposing the old HttpClient interface. The former didn't ultimately make things simpler. The latter is (I think) still a viable idea but I'd like to get the binary request/response stuff added before seeing what can be abstracted.

**Comment by EmmaLRussell** [comment]
That makes sense

> @EmmaLRussell Thanks! I did try to something along those lines: first by using a shared, part-completed `Builder` instance and then by repurposing the old HttpClient interface. The former didn't ultimately make things simpler. The latter is (I think) still a viable idea but I'd like to get the binary request/response stuff added before seeing what can be abstracted.

**Pull request merged**